src/ImageSharp/Advanced/AotCompilerTools.cs

```diff
@@ -107,8 +105,10 @@ private static void Seed<TPixel>()
         private static void AotCompileOctreeQuantizer<TPixel>()
             where TPixel : struct, IPixel<TPixel>
         {
-            var test = new OctreeFrameQuantizer<TPixel>(new OctreeQuantizer(false));
-            test.AotGetPalette();
+            using (var test = new OctreeFrameQuantizer<TPixel>(new OctreeQuantizer(false)))
```

Comment:
Spotted these when removing the AOT CreateDestination method. No longer required as not virtual.